src/components/PeopleFilters.tsx

```diff
@@ -1,12 +1,49 @@
+import classNames from 'classnames';
+import { useSearchParams } from 'react-router-dom';
+import { SearchLink } from './SearchLink';
+
 export const PeopleFilters = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+  const query = searchParams.get('query') || '';
+  const centuries = searchParams.getAll('centuries') || '';
+  const sex = searchParams.get('sex') || '';
+  const arrayCentury = ['16', '17', '18', '19', '20'];
```

Comment:
this should be outside of component 
```suggestion
  const FILTER_CENTURIES = ['16', '17', '18', '19', '20'];
```